src/colvargrid.h

```diff
@@ -1394,6 +1573,13 @@ class colvar_grid_gradient : public colvar_grid<cvm::real>
   /// Constructor from a multicol file
   colvar_grid_gradient(std::string &filename);
 
+  /// Constructor from a vector of colvars and a pointer to the count grid
+  colvar_grid_gradient(std::vector<colvar *> &colvars, colvar_grid_count *samples_in);
+
+  /// Parameters for smoothing data with low sampling
+  int full_samples;
```

Comment:
We can initialize plain old data in the header now! :-)

Reply:
Wouldn't the initialization list be even cleaner though? I think they should be gathered in one place.